### PR TITLE
Grid view favorites

### PR DESCRIPTION
**Pull request by Merkost**

### CODE REVIEW

File: app/src/main/res/menu/menu.xml [hunk 1]
```diff
@@ -34,6 +34,11 @@
         android:icon="@drawable/ic_info_vector"
         android:title="@string/about"
         app:showAsAction="ifRoom" />
+    <item
```
Comment (tibbi) [inline app/src/main/res/menu/menu.xml:37]: move this above settings and about, just like always

**Comment by tibbi** [comment]
allow changing the column count too, both with gestures and a menu item

**Comment by tibbi** [comment]
and allow the name be shown on 2 lines, with ellipsize in the end

**Comment by tibbi** [comment]
man, use the Column count menu item, like in the file manager.. Dont come up with custom solutions, keep the apps consistent

**Comment by Merkost** [comment]
> man, use the Column count menu item, like in the file manager.. Dont come up with custom solutions, keep the apps consistent

Oh, my bad, I missed that. Will do just that!

**Comment by Merkost** [comment]
Please, also check simple-commons pull request. https://github.com/SimpleMobileTools/Simple-Commons/pull/1735

**Comment by Merkost** [comment]
Do we need to extract these constants 
Screenshot 2023-07-14 at 10 33 58
as well as these config values to make it shared for contacts and dialer? 
Screenshot 2023-07-14 at 10 34 21

**Comment by tibbi** [comment]
merged. Also, make the range min-max 1-10. Move as much shared code into Simple Commons as possible.

**Comment by Merkost** [comment]
Alright! https://github.com/SimpleMobileTools/Simple-Commons/pull/1738

**Comment by Merkost** [comment]
All done here, can be tested now

**Comment by tibbi** [comment]
update to latest commons please

**Comment by tibbi** [comment]
font size setting doesnt seem to work, not sure if it is related to this issue though

**Comment by tibbi** [comment]
as it doesnt work anywhere

**Comment by Merkost** [comment]
> font size setting doesnt seem to work, not sure if it is related to this issue though

I'll check that issue👌

**Comment by Merkost** [comment]
> font size setting doesnt seem to work, not sure if it is related to this issue though

The issue was as rv adapters used previously initialised font size. Fixed.

**Comment by tibbi** [comment]
font size changing still doesnt work to me, it needs an app restart.. and changing the column count should have an animation like in our other apps, it is now instant

**Comment by Merkost** [comment]
> font size changing still doesnt work to me, it needs an app restart.. and changing the column count should have an animation like in our other apps, it is now instant


https://github.com/SimpleMobileTools/Simple-Dialer/assets/56008710/26b793a8-5939-4fba-9b40-b8a86f5ff009

Font size changing is working properly on my app, probably try build - rebuild project. I hope the will help.

**Comment by tibbi** [comment]
ok thanks, works now

**Pull request merged**